backend/app/models/__init__.py

```
@@ -0,0 +1,6 @@
+from app.models.audit_log import AuditLog
+from app.models.clinical_event import ClinicalEvent
+from app.models.encounter import Encounter
+from app.models.patient import Patient
+
+__all__ = ["Patient", "Encounter", "ClinicalEvent", "AuditLog"]
```

Ruff (0.16.0)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)